sympy/utilities/runtests.py

```diff
@@ -242,6 +250,10 @@ def run_all_tests(test_args=(), test_kwargs={}, doctest_args=(),
     """
     tests_successful = True
 
+    test_kwargs = test_kwargs or {}
```

Comment:
No, there is no reason, that I was explicit there but not here. I'll consolidat this. The only reason why I moved this into the function body is that it should be avoided to have mutable defaults in the function signature.